src/Common/tests/System/PlatformDetection.cs

```diff
@@ -23,6 +23,9 @@ public static class PlatformDetection
             File.Exists(Path.Combine(Environment.GetEnvironmentVariable("windir"), "regedit.exe")));
```

Comment:
Nano doesn't support any GUI programs, so used regedit as every other windows platform had with the previous requirements.